po/ka.po

```diff
@@ -0,0 +1,614 @@
+# Georgian translation for gnome-pomodoro
+# Copyright (C) 2024 gnome-pomodoro authors
+# This file is distributed under the same license as the gnome-pomodoro package.
+# Temuri Doghonadze <temuri.doghonadze@gmail.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-pomodoro\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-31 13:18+0200\n"
+"PO-Revision-Date: 2024-06-27 05:53+0200\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
+"Language-Team: Georgian <(nothing)>\n"
+"Language: ka\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.3.2\n"
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:8
+#: data/org.gnome.Pomodoro.desktop.in:3 data/resources/menus.ui:28
+#: data/resources/window.ui:382 plugins/actions/enums.vala:92
+#: plugins/gnome/extension/indicator.js:77
+#: plugins/gnome/extension/indicator.js:729
+#: plugins/gnome/extension/notifications.js:281
+#: plugins/gnome/extension/timer.js:41 src/about-dialog.vala:31
+#: src/main.vala:50 src/notifications-capability.vala:129
+#: src/stats-page.vala:527 src/window.vala:44
+msgid "Pomodoro"
+msgstr "პომოდორო"
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:9
+msgid "A time management utility for GNOME"
+msgstr "დროის მართვის პროგრამა GNOME-სთვის"
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:11
+msgid ""
+"A GNOME utility that helps managing time according to Pomodoro Technique. It "
+"intends to improve productivity and focus by taking short breaks after every "
+"25 minutes of work."
+msgstr ""
+"GNOME-ის პროგრამა, რომელიც დაგეხმარებათ, დრო პომოდოროს ტექნიკით მართოთ. ის "
+"ეცდება, გააუმჯობესოს თქვენი პროდუქტიულობა და ფოკუსი 25 წუთში ერთხელ "
+"დასვენებით შეთავაზებით."
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:23
+msgid "Timer window"
+msgstr "ტაიმერის ფანჯარა"
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:27
+#: plugins/gnome/extension/indicator.js:83 src/preferences-dialog.vala:800
+msgid "Preferences"
+msgstr "მორგება"
+
+#: data/org.gnome.Pomodoro.appdata.xml.in:31
+msgid "Indicator for GNOME Shell"
+msgstr "მაჩვენებელი GNOME-ის გარსისთვის"
+
+#: data/org.gnome.Pomodoro.desktop.in:4 src/about-dialog.vala:32
+msgid "A simple time management utility"
+msgstr "დროის მართვის მარტივი პროგრამა GNOME-სთვის"
+
+#: data/org.gnome.Pomodoro.desktop.in:5
+msgid "timer;"
+msgstr "ტაიმერი;"
+
+#: data/resources/menus.ui:6 plugins/actions/enums.vala:188
+#: plugins/actions/resources/action-page.ui:155 src/application.vala:90
+msgid "Start"
+msgstr "გაშვება"
+
+#: data/resources/menus.ui:11 data/resources/window.ui:180
+#: plugins/actions/enums.vala:197 plugins/actions/resources/action-page.ui:200
+#: src/application.vala:99
+msgid "Pause"
+msgstr "გაჩერება"
```

Comment:
Pause and Stop have same translation. How about using `პაუზა` here (suggestion from Google Translator)?